### ISSUE TITLE
What am I to make of this warning?

### ISSUE THREAD

**Issue by networks1** [fhamborg/NewsMTSC#33]
Is this something I should worry about? I have no idea what checkpoint RobertaModel is being initialized from. This is issued by the version installed with pip:

```
Some weights of the model checkpoint at roberta-base were not used when initializing RobertaModel: ['lm_head.layer_norm.bias', 'lm_head.decoder.weight', 'lm_head.dense.weight', 'lm_head.layer_norm.weight', 'lm_head.bias', 'lm_head.dense.bias']
- This IS expected if you are initializing RobertaModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing RobertaModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
```

**Comment by fhamborg** [comment]
this is fine. the same happens when you follow HF's instructions on their roberta-base page, https://huggingface.co/roberta-base

```
from transformers import RobertaTokenizer, RobertaModel
tokenizer = RobertaTokenizer.from_pretrained('roberta-base')
model = RobertaModel.from_pretrained('roberta-base')
text = "Replace me by any text you'd like."
encoded_input = tokenizer(text, return_tensors='pt')
output = model(**encoded_input)
```

in line 3 you will get the same warning. i suspect that roberta-base contains a head layer, more specifically a head layer consisting of `['lm_head.layer_norm.bias', 'lm_head.decoder.weight', 'lm_head.dense.weight', 'lm_head.layer_norm.weight', 'lm_head.bias', 'lm_head.dense.bias']`, which is however not used in NewsSentiment / NewsMTSC as we only use the plain `RobertaModel`

**Comment by fhamborg** [comment]
if you think this might be an issue, you should check with huggingface

**Comment by t1h0** [comment]
Fixed by ac57195a02d148f4dedc59e4f894a821d7a84dcd
